spring-cloud-azure-samples/eventhubs-kafka-sample/pom.xml

```diff
@@ -3,23 +3,50 @@
          xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>spring-cloud-azure-samples</artifactId>
         <groupId>com.microsoft.azure</groupId>
-        <version>1.2.0-SNAPSHOT</version>
+        <artifactId>spring-cloud-azure</artifactId>
+        <version>1.1.0.RC4</version>
```

Comment:
We should not change the version here. Is it because when 1.2.0-SNAPSHOT used, the project could not be compiled?